parachain/node/src/chain_spec.rs

```diff
@@ -111,6 +111,16 @@ fn testnet_genesis(
 			// Configure endowed accounts with initial balance of 1 << 60.
 			balances: endowed_accounts.iter().cloned().map(|k|(k, 1 << 60)).collect(),
 		}),
+		bridge: Some(BridgeConfig {
+			source_channels: SourceChannelConfig {
+				basic: SourceChannel {
+					address: hex!["2ffa5ecdbe006d30397c7636d3e015eee251369f"].into(),
```

Comment:
need to make these all dynamic

Reply:
actually its probably fine to leave hardcoded indefinitely:
 - works fine as is for local/testing
 - for ropsten+production, we should only need to do genesis once and so making it dynamic doesn't save too much time as hardcoded modification should be fine. maybe we can create a chain_spec_local.rs and chain_spec_ropsten.rs or something for then?

Reply:
@musnit the hard-coded chainspec can easily be overriden for production without having to edit and recompile that rust file:

```bash
# Dump generated spec:
target/release/artemis build-spec > config.json

# Edit spec and add custom config:
vim config.json

# Launch chain with custom config
target/release/artemis --chain config.json
```

In future though, as you say, we will want to add hardcoded specs for rococo/ropsten though. In which case, launching a chain with such a spec would be as simple as:

```bash
target/release/artemis --chain rococo-ropsten
```

@Rizziepit Yes! Though since our chain is trust-minimized, `sudo` is a no-go. So in practice we'd need to use democratic measures to vote on such changes. We'd need to incorporate the [Democracy](https://substrate.dev/docs/en/knowledgebase/runtime/frame#democracy) and [Elections](https://substrate.dev/docs/en/knowledgebase/runtime/frame#elections) for that though.